beacon/pow/src/main/java/tech/pegasys/teku/beacon/pow/TimeBasedEth1HeadTracker.java

```diff
@@ -41,7 +41,7 @@ public class TimeBasedEth1HeadTracker implements Eth1HeadTracker, RunLoopLogic {
   private final AsyncRunner asyncRunner;
   private final Eth1Provider eth1Provider;
 
-  private final Subscribers<HeadUpdatedSubscriber> subscribers = Subscribers.create(true);
+  private final Subscribers<HeadUpdatedSubscriber> subscribers = Subscribers.createWithReplay(true);
```

Comment:
What drove the need for this change?

Reply:
Notification is pushed before we subscribed, so we miss the heaf and when we are post-merge, new blocks on eth1 are only those that CL posts, which starts only when we discover enough peers and sync started (almost 1 minute every time for me on Kiln). So we miss actual head without changes and get next block (which is catched) only in a minute